README.md

```diff
@@ -39,6 +39,12 @@ Disabling proxy pages means you'll get a "Not Found" error if you try to access
 bundle exec rake
 ```
 
+### Running the app
+
+```
+./startup.sh
```

Comment:
This repeats the information in `### Before running the app`, but I agree that a `Running the app` section would be good to have!

I'd be tempted to just rename `Before running the app` to `Running the app`, and move this line of code to the top of that section. Then the rest of that section should explain how you'll hit GitHub API limits by default, so you'll either need a GITHUB_TOKEN env var or will need to use the SKIP_PROXY_PAGES option.